Core/Base/Client.py

```diff
@@ -92,7 +92,7 @@ def _getRPC(self, rpc=None, url='', timeout=600):
     if not rpc:
       if not url:
         url = self.serverURL
-      self.__kwargs.setdefault('timeout', timeout)
+      self.__kwargs['timeout'] = timeout
```

Comment:
This means that the `setTimeout` function is no longer useful, right?

Reply:
This fix was made because if a former call has set ’timeout’ to, say, 120, this value remains unchanged. Indeed, there is also the `setTimeout` method. What do you suggest?

Reply:
At least `@deprecated` the `setTimeout` function. it is not used in DIRAC as far as I can tell.